docs/asciinema/bzip-failure.cast

```diff
@@ -0,0 +1,119 @@
+{"version": 2, "width": 108, "height": 25, "timestamp": 1754758979, "env": {"SHELL": "/bin/bash", "TERM": "xterm-256color"}}
+[0, "o", "\u001b[H\u001b[2J\u001b[3J"]
+[0, "o", "\u001b[?2004h$ "]
+[3.934709, "o", "b"]
```

Comment:
The `eb ` is missing here somehow, seems easy enough to inject that back:
```suggestion
[1.934709, "o", "e"]
[2.534709, "o", "b"]
[3.134709, "o", " "]
[3.934709, "o", "b"]
```

Reply:
03bd6f4